woocommerce/woocommerce
# Ensure translation is loaded for new shared file in Checkout Block

In a recent PR https://github.com/woocommerce/woocommerce/pull/48010 we moved some shared code to a single file that we're now enqueuing, when loading the file, we marked its translation as false, and didn't load its translation, causing half translated Checkout. This PR fixes it for that file but we need more comprehensive E2E in a follow up.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Load up this PR after installing 9.2 translations.
2. Ensure all of Cart/Checkout are translated.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Ensure translation is fully loaded for certain parts of Checkout block.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
